3-enrich/README.md

```diff
@@ -16,6 +16,134 @@ The **Enrich** process takes raw Snowplow events logged by a [Collector][collect
 | [scala-common-enrich][e4]      | A shared library for processing raw Snowplow events, used in (1) and (3) | Production-ready |
 | [emr-etl-runner][e5]           | A Ruby app for running (1) and (2) on Amazon Elastic MapReduce           | Production-ready |
 
+## How to add an enrichment?
```

Comment:
I think a better place would be sce's readme?

Reply:
I was thinking that someone who is new to the pipeline might not know that a new enrichment has to be added in a library that is then used in jobs, and that the information should be found here. And some steps of the process include `spark-enrich`, not only sce. Do you still think that I should move it?